src/components/TodoForm.tsx

```diff
@@ -0,0 +1,49 @@
+
+import React from "react"
+
+function TodoForm(props: {
+        todos: any[]; onAdd: (arg0: {
+            // add 1 to current largest id
+            id: number; text: string; done: boolean;
```

Comment:
What you're essentially declaring here is a `Todo` type. We would have preferred to see this extracted into its own reusable type and used here instead. You'd then be able to use `Todo[]` instead of the `any[]` on line 5.

Reply:
Makes sense, it's a nice tip.